tests/lib/reset.sh

```diff
@@ -44,7 +59,11 @@ reset_classic() {
         systemctl start snapd.socket
 
         # wait for snapd listening
-        while ! printf "GET / HTTP/1.0\r\n\r\n" | nc -U -q 1 /run/snapd.socket; do sleep 0.5; done
+        EXTRA_NC_ARGS="-q 1"
+        if [[ "$SPREAD_SYSTEM" = fedora-* ]]; then
+            EXTRA_NC_ARGS=""
```

Comment:
Ah its coming via the `nmap` package on Ubuntu (https://packages.ubuntu.com/yakkety/nmap). That is why I didn't found it. Let me look into replacement `nc` here with `ncat` in another PR as there are more places than this one suffering from the same problem.